From 0f58d44c69e7c281531a2d16349d887bfeeb32d1 Mon Sep 17 00:00:00 2001
From: lijuan <juan.li@spacemit.com>
Date: Wed, 25 Dec 2024 14:52:21 +0800
Subject: [PATCH 1397/1448] pcie: fix build warning

1. WARNING: modpost: vmlinux: section mismatch in reference: k1x_pcie_driver+0x0 (section: .data) -> k1x_pcie_probe (section: .init.text)

Change-Id: I3a9cd3656bbeeb956cc6b1518f68391fa2715e17
Signed-off-by: lijuan <juan.li@spacemit.com>
---
 drivers/pci/controller/dwc/pcie-k1x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/dwc/pcie-k1x.c b/drivers/pci/controller/dwc/pcie-k1x.c
index 2c9ee0359e29..ace9408adba6 100644
--- a/drivers/pci/controller/dwc/pcie-k1x.c
+++ b/drivers/pci/controller/dwc/pcie-k1x.c
@@ -1563,7 +1563,7 @@ static int k1x_power_on(struct k1x_pcie *k1x, int on)
 	return 0;
 }
 
-static int __init k1x_pcie_probe(struct platform_device *pdev)
+static int k1x_pcie_probe(struct platform_device *pdev)
 {
 	u32 reg;
 	int ret;
-- 
2.47.0

